Source/Core/VideoCommon/RenderState.cpp

```diff
@@ -179,6 +179,21 @@ void BlendingState::Generate(const BPMemory& bp)
       }
     }
   }
+
+  // If we aren't writing color or alpha, don't blend it.
+  // Intel GPUs on D3D12 seem to have issues with dual-source blend if the second source is used in
+  // the blend state but not actually written (i.e. the alpha src or dst factor is src alpha, but
+  // alpha update is disabled).
+  if (!colorupdate)
+  {
+    srcfactor = SrcBlendFactor::One;
+    dstfactor = DstBlendFactor::Zero;
+  }
+  if (!alphaupdate)
+  {
+    srcfactoralpha = SrcBlendFactor::One;
+    dstfactoralpha = DstBlendFactor::Zero;
```

Comment:
On further thought, this probably isn't the best choice of values; if updating is disabled, it would make more sense to set the source factor to zero and the dest factor to 1, i.e. the new color in the framebuffer is 0*the tev's output + 1 * what's already in the framebuffer.

Of course, since updating is disabled, this shouldn't matter at all, but it would make things a bit more obvious e.g. when looking at the blend state in renderdoc.